analyzers/its/Directory.Build.targets

```
@@ -52,8 +52,8 @@
       <!-- Remove all previously added analyzers, except built-in SourceGenerators -->
       <Analyzer Remove="@(Analyzer)" Condition="!$([System.Text.RegularExpressions.Regex]::IsMatch('%(Filename)', '(Generator|Generators|SourceGeneration|SourceGenerators)$'))"/>
       <!-- Add the SonarAnalyzer analyzer DLLs -->
-      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer*.dll" />
-      <Analyzer Include="$(BinariesFolder)\Google.Protobuf.dll" />
+      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer.*\SonarAnalyzer*.dll" />
+      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer.*\Google.Protobuf.dll" />
```

The new structure actually enables us to fix this old workaround. It should name the directory explicitly, and take `*.dll` files from it
```suggestion
      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer.CSharp\*.dll" />
      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer.VisualBasic\*.dll" />
```